### DISCUSSION TITLE
Vacation update - Back October 18th

### DISCUSSION THREAD

**Discussion by Ylianst** [Ylianst/MeshCentral#4586] [Announcements]
Hi all. I just wanted to just in a say hi. As you know, I am on vacation and traveling until October 18th 2022. So, I still have 3 weeks before I am back. I am doing great and look forward to starting up work again, lots of bug reports and suggestions to work on. I am in the south of France now, totally enjoying it. The weather is great and doing a LOT of tourist stuff.

**Comment by nicrame** [comment]
Cool! Have a good trip! And welcome in Europe! :)

**Comment by Nikocode** [comment]
You rock! Hope the weather stays pleasant and you have a wonderful time!

**Comment by mattpogue** [comment]
Thank you for giving an update. I hadn't seen your previous message and I've been so used to the frequent version updates that I was afraid something might have happened to you lol. You do a tremendous job with this project and MeshCentral is an indispensable support tool for me. I hope you have a fantastic and relaxing time and thank you again for the work you do!

**Comment by Ylianst** [comment]
One more update. In the last few weeks I have been to France, Italy, Monaco, Spain and Andorra. Just left Andorra today, did not even know about this country until a few days ago, it's a crazy mix between a ski resort and a shopping mall, I loved it. It's the 8th now, I am back in 10 days. Can't wait to get started again.

**Reply by silversword411** [reply]
Don't you have a public photo gallery? What's the link?

Sounds like a vicarious living opportunity for the rest of us! 🙂

**Reply by Ylianst** [reply]
I will work on a small photo album, for now, I have a map of my vacations. It starts and ends in Paris. Goes thru 4 countries, I went clockwise.

![France-Map-1](https://user-images.githubusercontent.com/1319013/196415992-02ca88a5-db1a-4f8c-89bb-d6ae76ee1979.png)

**Reply by dinger1986** [reply]
looks fantastic! Alot of lovely areas around there and similar to some trips I have taken in the past all be it substantially shorter

**Comment by Ylianst** [comment]
Quick update. I am flying back to Portland Oregon today!! After 6 weeks, I finally come home and get to code again. I will have a lot to catch up on both on GitHub and with my corporate work. I will probably be jet lagged like crazy. I had a wonderful trip and saw some many sites and have so many stories, it's crazy. I would not know where to start.

Apologies for everyone that asked questions or made requests and did not get responses, I will try to catch up as quickly as possible.

**Comment by Ylianst** [comment]
Since I have a few minutes, here is my wife and I being super nerds at 2 sites I visited. 2000 years of bridges. Left is [Pont du Gard](https://en.wikipedia.org/wiki/Pont_du_Gard) is the tallest of all Roman [aqueduct bridges](https://en.wikipedia.org/wiki/Aqueduct_(bridge)), right is the [Millau Viaduct](https://en.wikipedia.org/wiki/Millau_Viaduct), as of September 2020, it is the [tallest bridge in the world](https://en.wikipedia.org/wiki/List_of_tallest_bridges),

![2000-years-of-bridges](https://user-images.githubusercontent.com/1319013/196418585-38f37b82-4c96-4c38-9f7d-c362c29f802c.jpg)
